tox.ini

```diff
@@ -1,330 +1,224 @@
 [tox]
-envlist = py{27,33,34,35,36,37,38}, py{35,36,37,38}-travis, docs, flake
+envlist = py{310,311,312,313}, docs, flake
 skip_missing_interpreters=True
 
 
 [base]
 passenv =
-	CODECOV_TOKEN
-	DEBIAN_FRONTEND
-	CONTINUOUS_INTEGRATION
-	PYTHON*
-	CI
-	CI_*
-	GIT_*
-	TRAVIS
-	TRAVIS_*
-	VERSIONER_PYTHON_VERSION
-deps = -r{toxinidir}/requirements.txt
+    CODECOV_TOKEN
+    CODECOV_ENV
+    DEBIAN_FRONTEND
+    CONTINUOUS_INTEGRATION
+    PYTHON*
+    CI
+    CI_*
+    GITHUB_*
+    GIT_*
+    TRAVIS
+    LC_TYPE
+    LANG
+    TRAVIS_*
+    VERSIONER_PYTHON_VERSION
+    COV_CORE_*
+    COVERAGE_*
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/tests/requirements.txt
+    -r{toxinidir}/docs/requirements.txt
 
 
-[testenv:py{27,33,34,35,36,37,38}]
+[testenv:py{310,311,312,313}]
 sitepackages = False
+skipsdist = False
 recreate = True
 alwayscopy = True
 passenv =
-	{[base]passenv}
+    LC_ALL="en_US.utf8"
+    {[base]passenv}
 basepython =
-	py27: python2.7
-	py33: python3.3
-	py34: python3.4
-	py35: /usr/bin/python3.5
-	py36: /usr/bin/python3.6
-	py37: python3.7
-	py38: python3.8
+    py310: python3.10
+    py311: python3.11
+    py312: python3.12
+    py313: python3.13
 deps =
-    coverage>=4.4
-    codecov>=2.0.9
+    pip>=22.0
+    coverage>=6.3
+    codecov>=2.0.15
     {[base]deps}
 commands =
-	py27: python2.7 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py33: python3.3 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py34: python3.4 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py35: python3.5 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py36: python3.6 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py37: python3.7 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	py38: python3.8 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-
-
-[testenv:py27]
-sitepackages = False
-recreate = True
-alwayscopy = True
-passenv =
-	TRAVIS_PYTHON_VERSION="2.7"
-	{[base]passenv}
-basepython = python2.7
-deps =
-    coverage>=4.4
-    codecov>=2.0.9
-    {[base]deps}
-commands = 
-	coverage erase
-	python2.7 -m coverage run -p --source=pak -m unittest discover --verbose -t ./
-	- coverage combine
-	- coverage report --include=pak*
-	- codecov
+    py310: python3.10 -m coverage run -p --source=pypak -m unittest discover --verbose -t ./
+    py311: python3.11 -m coverage run -p --source=pypak -m unittest discover --verbose -t ./
+    py312: python3.12 -m coverage run -p --source=pypak -m unittest discover --verbose -t ./
+    py313: python3.13 -m coverage run -p --source=pypak -m unittest discover --verbose -t ./
```

Comment:
"pypak" is basically shorthand for "Python-PAK" here, and it improves readability in the documentation to drop the hyphen. I do agree that normally, the consistency would be preferred, especially considering this change was to avoid a name-collision.